Makefile

```diff
@@ -189,10 +191,14 @@ image-build: build test ## Build container image with the manager.
 image-push: ## Push container image with the manager.
 	${CONTAINER_ENGINE} push ${IMG}
 
+.PHONY: image-fips-scan
+image-fips-scan: image-build image-push
```

Comment:
*nit* Do you need to do build and push in this step?

I had a bit of pain with this, because to build and push I use different registry auth files (one that can pull the images from registry.ci.openshift.org, another that can push to quay.io under my user).

So I had to work around this by doing the build and push ahead of time.

Maybe it's just my particular case was painful, so it's not that big of a deal.

Reply:
> Do you need to do build and push in this step?

Yes, `check-payload` needs the image to be pullable.

> I had a bit of pain with this, because to build and push I use different registry auth files (one that can pull the images from registry.ci.openshift.org, another that can push to quay.io under my user).

Interesting. I'm logged to my quay.io using `podman login`. The CI image registry auth is saved to `/run/user/${id}/containers/auth.json` thanks to `oc registry login` command. And they don't interfere on my machine.

Reply:
Yea this sounds like a personal problem I have with managing podman auth files. I should combine them.